pages/blog/typescript-types.mdx

```diff
@@ -0,0 +1,145 @@
+import withArticle from '../../ui/blog/article'
+export const meta = {
+  title: 'Back to front: TypeScript union types and how to use GraphQL Codegen to create type-safe',
```

Comment:
Hmm I'm not sure I understand the title

Reply:
Charly suggested a new title:

`Getting the best of TypeScript and GraphQL: union types`